### ISSUE TITLE
[Bug]: Server bug started occuring seemingly out of nowhere- "Packet 1/0 (ServerboundStatusRequestPacket) was larger than I expected"

### ISSUE THREAD

**Issue by HenrikoMagnifico** [someaddons/connectivity#57]
### Describe the bug you're experiencing

Seemingly out of nowhere our server stopped working, possibly because of an error getting spammed in chat that reads as; 
```
[Netty Epoll Server IO #2/WARN] [co.co.Connectivity/]: io.netty.handler.codec.DecoderException: java.lang.ArrayIndexOutOfBoundsException: Index 13 out of bounds for length 2
[Netty Epoll Server IO #2/WARN] [co.co.Connectivity/]: io.netty.handler.codec.DecoderException: java.lang.IndexOutOfBoundsException: Index 56 out of bounds for length 51
```
This makes it impossible to join the server from any client. It gets to the "encrypting" stage when trying to join, but gets stuck there and gets no further, resulting in a "timed out" message.  The server has been running fine for weeks up until this point. I have no idea why it started happening. Both the latest.log and debug.logs have been attached below.

Connectivity version: connectivity-1.19.2-3.4
Forge version: 43.2.1
Java version: 17

[debug.log](https://github.com/someaddons/connectivity/files/10951037/debug.log)

### Reproducability

Use connectivity-1.19.2-3.4 with the other included mods on a server. Play for a long while and the error might occur. Very hard to reproduce(?)

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Minecraft version

1.19

### Modloader version

Forge

### Logs

https://gist.github.com/HenrikoMagnifico/acb0618cd3f3aac98e3f0a0fa36f2ac7

**Comment by someaddons** [comment]
must be one of your other mods causing it, connectivity just displays the error in the log

**Comment by someaddons** [comment]
fixed in 3.5
